src/Samhammer.Logging.Logstash/Samhammer.Logging.Logstash.csproj

```diff
@@ -1,28 +1,33 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
@datlam162 we should keep targetframework netstandard2.0 when possible.
cause they can be used in .NET 4.6.x also and not only netcore.
we only need to change to net6.0 when we need frameworkreferences to aspnet